extensions/wikia/FounderEmails/events/FounderEmailsEditEvent.class.php

```diff
@@ -318,23 +318,15 @@ public static function recentChanges( $oRecentChange ) {
 			$wasNotificationSent = ( static::getFirstEmailSentFlag( $editor->getName() ) === '1' ) ;
 
 			if ( !$wasNotificationSent ) {
-				$userEditStatus = static::getUserEditsStatus( $editor, true );
```

Comment:
This method would try to calculate edit count of user from master DB. However, we don't care about the actual edit count—we just want to know if it is 0. If it's 0, then it means the user is making their first edit now. So we can use the existing `UserStatsService` to give us that data, which doesn't involve reading from master.